astronomer/providers/amazon/aws/operators/emr.py

```diff
@@ -39,6 +42,13 @@ def execute(self, context: Context) -> None:
             configuration_overrides=self.configuration_overrides,
             client_request_token=self.client_request_token,
         )
+        try:
+            # for apache-airflow-providers-amazon<6.0.0
+            polling_attempts = self.max_tries  # type: ignore[attr-defined]
+        except AttributeError:  # pragma: no cover
+            # for apache-airflow-providers-amazon>=6.0.0
+            # max_tries is deprecated so instead of max_tries using self.max_polling_attempts
+            polling_attempts = self.max_polling_attempts  # type: ignore[attr-defined]
```

Comment:
```suggestion
            polling_attempts = self.max_polling_attempts
```
We won't need this once the RC is released